src/main/java/com/softserveinc/dokazovi/aop/PostLogger.java

```diff
@@ -90,8 +92,12 @@ public void deletePost(JoinPoint joinPoint) {
         if (!flag) {
             return;
         }
-        PostEntity postEntity = postRepository.getOne(postId);
-        makeEntryInLogs(postEntity.getTitle(), userPrincipal, "Матеріал видалено", null);
+        PostTitleDTO postTitleDTO = postDeleteEventHandler.getPostTitleDTO();
```

Comment:
Move this to the event listener. If needed, put the logic into a service which encapsulates that.
It really does not belong here, as it is a side effect, not the main one.